website/docs/hyper-api/hyper_process.md

```diff
@@ -260,12 +258,24 @@ To create a new Hyper database file with this version, set
 :::note
 The database file format version `3` is supported by Tableau Desktop
 2022.4.1 and Server 2023.1 and newer product versions. It is supported by
-Tableau Prep 2022.4.1 and newer versions. Opening a database file with an
-unsupported Tableau product version will produce an error message similar to:
+Tableau Prep 2022.4.1 and newer versions. 
+:::
 
-"There was an error during loading database '[...]/file.hyper':
-unsupported version 3 (max supported version: 2). To open this database,
-please update your product. (error code 0AS01)"
+#### version 4
+Database file format version `4` was introduced to support
+persisting and reading the new 32-bit floating point type.
+
+Starting with release (#TODO), Hyper uses 32-bit floats for
+the SQL types `real` and `float(p)` with `p <= 24`.
+The types `double precision`, `float`, and `float(p)` with `p >= 25`
+still use 64-bit doubles.
```

Comment:
Do we want to mention `float4` and `float8` here?

Reply:
c150724ae4adfce1b6eaee1d2c8c22ae568ad06a